designer/server/src/main/scala/pl/touk/nussknacker/ui/api/MigrationApiEndpoints.scala

```diff
@@ -0,0 +1,159 @@
+package pl.touk.nussknacker.ui.api
```

Comment:
Now we keep these classes inside api/description

Reply:
Done